CHANGELOG.md

```diff
@@ -7,6 +7,8 @@
 #### UWP/WPF/WinForms
 
 * **[Fix]** Fix an issue where internal SDK exceptions would sometimes be "unobserved" and would cause undefined behavior.
+* **[Fix]** Fix deleting the database file and opening a new one whenever corruption is detected.
```

Comment:
the `Fix` above this line states what is being fixed, but this line says `Fix`, then describes what the fix is - but it reads like the deleting and opening a new one is the bug being fixed.

Reply:
I'll try a different wording.